Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -132,6 +132,10 @@ class BrowserViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override var prefersStatusBarHidden: Bool {
+        return false
```

Comment:
Probably don't want this in the BVC class, it will hide the status bar all the time.

Reply:
@garvankeeley I'm not sure if I correctly understood your point, but this is to show the status bar when it's in the BVC class. The Intro Class has the opposite behaviour (hiding all the time).

Reply:
Sorry! I read this backwards.

Reply:
@garvankeeley no problem, should I change something or it's ok?